app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -217,6 +217,10 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
   const innerHintId = `${hintId}-inner`;
   const labelId = `${inputId}-label`;
   const showInnerHint: boolean = !value && !isValuePending && !isMobile;
+  // In test only we allow the upload of yaml files, but because they're text files
+  // they don't have a preview. This shows the name of the file in the upload
+  // box (using the existing preview) when the file name ends with .yml
+  const isYamlFile: boolean = value instanceof window.File && value.name.endsWith('.yml');
```

Comment:
Nit: In JavaScript, conventional naming would capitalize "YAML".

Ref: https://github.com/18F/identity-idp/blob/main/docs/frontend.md#naming-conventions

>All letters of an acronym should be capitalized

```suggestion
  const isYAMLFile: boolean = value instanceof window.File && value.name.endsWith('.yml');
```

Reply:
Ah, thanks. Fixed that [here](https://github.com/18F/identity-idp/pull/10134/commits/82b3ffa7a175f88b32c3fdd669ea2d7912805e99).